api.md

```diff
@@ -2775,6 +2775,8 @@ Specifically this sets the `timescaledb.restoring` GUC to `on` and stops any
 background workers which may have been performing tasks until the [`timescaledb_post_restore`](#timescaledb_post_restore)
 fuction is run following the restore. See [backup/restore docs][backup-restore] for more information.
 
+>:WARNING: Using this function when doing an upgrade could cause issues in previous versions of TimescaleDB.
```

Comment:
What does  "previous versions" mean?  Should we specify anything like "versions earlier than 1.x"?

Reply:
Clarified.